main/updater.js

```diff
@@ -54,7 +54,7 @@ module.exports = async function setupUpdater (
   /** @type {import('./typings').Context} */ ctx
 ) {
   ctx.getUpdaterStatus = function getUpdaterStatus () {
-    return { updateAvailable }
+    return { updateAvailable: updateDownloaded }
```

Comment:
I would prefer to rename the places reading this property too. Otherwise, we are using different names ("update available" vs "update downloaded") for the same thing and this could create confusion in the future.

Reply:
I was thinking about this too, but decided against this because I think the places outside this library shouldn't have to care about the finer details of updating. I could see however how `updateDownloaded` is more clear. I was just thinking that as an outsider, "update downloaded" could be more confusing than "update available" (ok it's downloaded, are there any other steps?). Maybe instead something like `readyToUpdate`?

Reply:
what do you think about this?

Reply:
I agree with your reasoning. 👍🏻 

I like the idea of renaming the property to `readyToUpdate` or perhaps `updateReady`.